### DISCUSSION TITLE
API key / Token-based auth: prefix

### DISCUSSION THREAD

**Discussion by Theodlz** [scalar/scalar#3343] [Ideas]
Lots of APIs have token-based authentication where the header looks like:
```python
headers = {'Authorization': f'token {token}'}
```

It would be awesome if we could specify a prefix, as a simple string parameter (e.g. "bearer", or "token", ...).

That way, the user just has to input the token in the text input that already exists (that the top right of the docs), and the autogenerated code that shows up for each endpoint would be correctly formatted, with said prefix in front of the user's token.

**Comment by hanspagel** [comment]
Issue: https://github.com/scalar/scalar/issues/3687
